File: DAY3/sample3.cpp
// sample3.cpp
class A 
{
public:
	void cry() {}
};
class B
{
public:
	void cry() {}
};
// 멤버 함수 cry 를 가진 모든 객체를 인자로 받고 싶다.
void foo(A c) { c.cry();}

int main()
{
	A a;
	B b;
	foo(a);
	foo(b);
}

File: DAY3/sample3.py
class A:
	def cry(self):
		pass
class B:
	def cry(self):
		pass

def foo(x):	# 타입이름 없이 변수명(x) 만 표기
	x.cry()

a = A()
b = B()
foo(a)
foo(b)

